src/main/java/author/AuthorList.java

```diff
@@ -2,20 +2,20 @@
 
 import java.util.ArrayList;
 
-public class AuthorList {
-    private ArrayList<Author> authorList;
-
-    public AuthorList() {
-        this.authorList = new ArrayList<>();
-    }
-
+public class AuthorList extends ArrayList<Author> {
```

Comment:
Appreciate the change to directly extend instead of our previous wrapper class, this should make working with `AuthorList` more straightforward.